apps/dashboard/forms.py

```diff
@@ -64,6 +65,10 @@ class DashboardFilterForm(forms.Form):
         widget=forms.SelectMultiple(),
     )
 
+    participants = forms.MultipleChoiceField(choices=[], required=False, widget=forms.SelectMultiple)
+
+    tags = forms.MultipleChoiceField(choices=[], required=False, widget=forms.SelectMultiple())
```

Comment:
You can make these `ModelMultipleChoiceField` and follow the same pattern as experiments.